### PR TITLE
HacktoberFest TeaTime logo to readme

### PR DESCRIPTION
**Pull request by thehashton**
Adds Tea Time logo to readme for Hacktoberfest!! :)

+ tidies up the readme a bit

- Harry

### CODE REVIEW

**Comment by AndreVarandas** [comment]
Hi @hghazni thanks for your pull request! We have accepted one logo, do you still want to submit your linting changes + reference to the new logo that is now in the root of the project?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+<p align="center">
+<img width="250" height="250" src="https://i.imgur.com/jDl9FW8.png">
```
Comment (AndreVarandas) [inline README.md]: Here, we can link to the existing logo on the root of the project 😃

**Review by AndreVarandas — CHANGES_REQUESTED** [review]
It would be great also to move the logo on the root of the folder to the suggested  src/assets/images/teatimelogo.png 👍

File: README.md (hunk 1 above)
Comment (thehashton) [reply README.md]: Done @AndreVarandas :)

**Comment by thehashton** [comment]
I do still want to submit it :)

**Review by AndreVarandas — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AndreVarandas) [reply README.md]: Thank you for your time and patience! 🚀

**Pull request merged**